runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -223,12 +238,94 @@ public void onChatMessage(ChatMessage chatMessage)
 		}
 	}
 
+	@Subscribe
+	public void onChatboxInput(ChatboxInput chatboxInput)
+	{
+		final String value = chatboxInput.getValue();
+		if (!value.startsWith("!kc "))
+		{
+			return;
+		}
+
+		chatboxInput.setStop(true);
+
+		final String boss = longBossName(value.substring(4));
+
+		final int kc = getKc(boss);
+		if (kc <= 0)
+		{
+			chatMessageManager.queue(QueuedMessage.builder()
```

Comment:
I dont agree with not resuming the message and simply not sending them to other players. 1. that would look weird, when player sent some message to chat but nothing got displayed on other side 2. we dont do this in any other command and we just simply send it and display error message to everyone